### PR TITLE
ACP-2930: Added a notification about necessity of App disconnection on destructive deployment

### PR DESCRIPTION
**Pull request by pushokwhite**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/ratings-reviews/202404.0/third-party-integrations/configure-bazaarvoice.md [hunk 1]
```diff
@@ -78,3 +78,7 @@ This adds the Bazaarvoice app to your store. It usually takes Bazaarvoice a few
 You can do the administration work on the Bazaarvoice reviews from the [Bazaarvoice portal](https://portal.bazaarvoice.com/signin?ref=spryker-documentation). For example, you can approve individual reviews. See [Workbench overview](https://knowledge.bazaarvoice.com/wp-content/brandedge-pro-wb/en_US/basics/workbench_overview.html#log-in-to-workbench?ref=spryker-documentation) for details on how you can manage reviews from the Bazaarvoice portal.
 
 {% endinfo_block %}
+
+{% info_block warningBox "Disconnect Bazaarovice before running destructive deployment" %}
+[Destructive deployment](https://spryker.com/docs/dg/dev/acp/acp-destructive-deployment.html) permanently deletes all the existing Bazaarvoice configurations.
```
[the remark this answers is not in the document]
Comment (pushokwhite) [reply docs/pbc/all/ratings-reviews/202404.0/third-party-integrations/configure-bazaarvoice.md:83]: Assets, CmsSlot

[the remark this answers is not in the document]
Comment (matweew) [reply docs/pbc/all/ratings-reviews/202404.0/third-party-integrations/configure-bazaarvoice.md:83]: I think the term `configurations` does not fit in this case. As for a user it is not clear for me what will be deleted

[the remark this answers is not in the document]
Comment (pushokwhite) [reply docs/pbc/all/ratings-reviews/202404.0/third-party-integrations/configure-bazaarvoice.md:83]: That's why I summarized these things in one word. The user may not know what exactly will be deleted, but the fact of deletion should be clearly visible.

File: docs/dg/dev/acp/acp-destructive-deployment.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+With ACP, you need to consider the following when running destructive deployment:
+* Every connected app might have already created resources for your project.
+* Wiping out the resources of an app may lead to data loss or inconsistency.
+
+To avoid these issues, you need to follow the best practices when using ACP and destructive deployment together:
+
+{% info_block warningBox "Disconnect every connected App before running destructive deployment" %}
+Before running destructive deployment, disconnect every connected App from the App Catalog.
+This ensures that the App can recreate the resources from scratch.
+If additional steps are required to disconnect an App, you will receive a warning message.
```
[the remark this answers is not in the document]
Comment (pushokwhite) [reply docs/dg/dev/acp/acp-destructive-deployment.md]: Added

**Pull request merged**